components/brave_vpn/browser/connection/win/brave_vpn_helper/brave_vpn_helper_crash_reporter_client.cc

```diff
@@ -54,12 +55,11 @@ void BraveVPNHelperCrashReporterClient::InitializeCrashReportingForProcess(
   }
   install_static::InitializeProductDetailsForPrimaryModule();
   crash_reporter::SetCrashReporterClient(instance);
-  std::wstring user_data_dir;
-  install_static::GetUserDataDirectory(&user_data_dir, nullptr);
+  std::wstring vpn_data_dir = brave_vpn::GetVpnServiceProfileDir().value();
```

Comment:
nit: it seems we don't need to make this var? Instead calling `brave_vpn::GetVpnServiceProfileDir().value()` directly below seems better.

Reply:
done